src/reference/sectionC_interpretation.inc

```diff
@@ -167,33 +167,38 @@ Interpretation of ``units`` elements
+
       Tuples for each :code:`unit` child element SHALL be determined as follows:
 
-      1. Where the :ref:`units reference<specC_units_reference>` of the :code:`unit` child element is to a single unit which is an irreducible unit, then the set of tuples SHALL have a single member, which SHALL consist of the name of the irreducible unit being referenced and the exponent 1.0.
+      1. If the :ref:`units reference<specC_units_reference>` of the :code:`unit` child element is to a single unit which is an irreducible unit, then the set of tuples SHALL have a single member, which SHALL consist of the name of the irreducible unit being referenced and the exponent 1.0.
 
-      2. Where the units reference of the :code:`unit` child element is to built-in units other than an :ref:`irreducible unit<specC_irreducible_units>`, then the tuples SHALL be derived directly from the :ref:`Built-in units<table_built_in_units>` table.
+      2. If the units reference of the :code:`unit` child element is to built-in units other than an :ref:`irreducible unit<specC_irreducible_units>`, then the tuples SHALL be derived directly from the :ref:`Built-in units<table_built_in_units>` table.
          Specifically, the set of tuples SHALL consist of the tuples given in the "Unit reduction tuple set" column of the row for which the value in the "Name" column matches the name of the units reference.
 
-      3. Where the units reference of the :code:`unit` child element is to a unit which is neither built-in, nor an irreducible unit, the set of tuples SHALL be defined recursively as the set of tuples for the :code:`units` element so referenced.
+      3. If the units reference of the :code:`unit` child element is to a unit which is neither built-in, nor an irreducible unit, then the set of tuples SHALL be defined recursively as the set of tuples for the :code:`units` element so referenced.
 
       4. The exponents of each tuple in the set for the current :code:`unit` element, as derived by following rules :hardcodedref:`3.3.3.2.1, 3.3.3.2.2 or 3.3.3.2.3` above, SHALL be multiplied by the exponent term of the current, referencing, :code:`unit` element.
 
-   3. Tuples which have the name element of "dimensionless" SHALL be removed from the set of tuples.
+   3. Tuples which have the unit name of "dimensionless" SHALL be removed from the set of tuples.
       Note that this can result in the set of tuples being empty.
 
-   4. Where the set of tuples consists of tuples which have the same name element, those tuples SHALL be combined into a single tuple with that name element and an exponent being the sum of the combined tuples’ exponents.
-      If the resulting tuple’s exponent term is zero, the tuple SHALL be removed from the set of tuples.
+   4. If the set of tuples contains tuples which have the same unit name, then those tuples SHALL be combined into a single tuple with that unit name and an exponent being the sum of the combined tuples’ exponents.
```

Comment:
Do we need "sum" as well as "combined"?  
Suggest: "... an exponent being the sum of those tuples' exponents."

Reply:
Agreed with @kerimoyle's suggestion.

Reply:
I think having combined in there is more clear

Reply:
There's a full set of tuples, and this bit is about the subset of "combined" tuples with the same name, so worth keeping that in there to distinguish?
As in, it's not the sum of _all_ tuples exponents, just of the ones with the same name.

Reply:
Yep, hence the those: "sum of those tuples' exponents"

Reply:
ok changed it